### DISCUSSION TITLE
Make Model Cards "diff-able"

### DISCUSSION THREAD

**Discussion by hanneshapke** [tensorflow/model-card-toolkit#232] [Ideas]
It would be great if the model card can be produced in an easy diffable output format. to easily compare two different models or model versions.

**Comment by codesue** [comment]
It's possible to output model cards in Python dictionary, JSON, and proto formats by calling `model_card.to_dict()`, `model_card.to_json()`, and `model_card.to_proto()`. The functionality is implemented in [base_model_card_field.py](https://github.com/tensorflow/model-card-toolkit/blob/master/model_card_toolkit/base_model_card_field.py) and [model_card.py](https://github.com/tensorflow/model-card-toolkit/blob/master/model_card_toolkit/model_card.py#L454). There are some usage examples in the [ModelCardToolkit](https://github.com/tensorflow/model-card-toolkit/blob/master/model_card_toolkit/core.py#L53) class and in tests, but I don't believe this is documented.

**Reply by codesue** [reply]
Hi @hanneshapke, does this solution work for you?

**Reply by hanneshapke** [reply]
Wow, `model_card.to_json()` would pave the way to use something like https://github.com/seperman/deepdiff#ignore-order
(no endorsement of the package :))
